src/components/Home.jsx

```diff
@@ -57,6 +75,106 @@ function Home() {
           <p>{homeData?.hours_teaching} + hours of teaching</p>
         </div>
       </div>
+      <div className="card">
+        <HStack spacing="24px" mt={10}>
+          <Card
+            w="280px"
+            height="420px"
+            rounded="20px"
+            overflow="hidden"
+            bg="gray.700"
+            mt={10}
+          >
+            <Image src={logoInverted} alt="Card Image" boxSize="210px"></Image>
+            <Box p={5}>
+              <Stack align="center">
+                <Badge variant="solid" color="teal.100" rounded="full" px={2}>
+                  RECursion
+                </Badge>
+              </Stack>
+              <Stack align="center">
+                <Text as="h2" fontWeight="normal" my={2}>
+                  Mission
+                </Text>
+                <Text fontWeight="light">
+                  Working towards the improvement of campus&apos;s coding
+                  culture by organizing regular coding classes and contests.
+                </Text>
+              </Stack>
+              <Flex>
+                <Spacer />
+                <Button variant="solid" color="teal.200" size="sm">
+                  Learn More
+                </Button>
+              </Flex>
+            </Box>
+          </Card>
+          <Card
+            w="280px"
+            height="420px"
+            rounded="20px"
+            overflow="hidden"
+            bg="gray.700"
+            mt={10}
+          >
+            <Image src={logoInverted} alt="Card Image" boxSize="210px"></Image>
+            <Box p={5}>
+              <Stack align="center">
+                <Badge variant="solid" color="teal.100" rounded="full" px={2}>
+                  RECursion
```

Comment:
Change the 'Badge' to Mission / Vission / Value and make the font little larger.

Reply:
ok